exporter/sentryexporter/interfaces.go

```diff
@@ -34,6 +39,23 @@ type SentrySpan struct {
 	StartTimestamp time.Time `json:"start_timestamp"`
 	EndTimestamp   time.Time `json:"timestamp"`
 	Status         string    `json:"status"`
+	LibName        string    `json:"-"`
+	LibVersion     string    `json:"-"`
+	ResourceTags   Tags      `json:"-"`
+}
+
+// MarshalJSON converts the SentrySpan struct to JSON.
+func (s SentrySpan) MarshalJSON() ([]byte, error) {
+	type alias SentrySpan
+	return json.Marshal(&struct {
+		StartTimestamp time.Time `json:"start_timestamp,omitempty"`
```

Comment:
```suggestion
		StartTimestamp time.Time `json:"start_timestamp"`
```

`omitempty` doesn't apply to structs.